pkgs/fake_async/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Does it matter that this is not our standard license?

Reply:
@devoncarew ?

Reply:
Let's keep the existing license as part of the move. I believe we can re-license it - into our std. license - but for here let's just do a mechanical move.